packages/wallets/src/components/Base/Divider/Divider.scss

```diff
@@ -0,0 +1,9 @@
+.wallets-divider {
+    &__horizontal {
```

Comment:
can we make it more generics with top, bottom, left, right variants maybe? in case we need it. So horizontal is for left + right border and vertical is for top + bottom border.